docs/community/project-showcase.md

```diff
@@ -21,10 +21,31 @@ Please make sure your project is well documented, including a README on how to r
 }
 [/block]
 
+
 This is the first real-world demo of Monty the TBP team came up with. We used the iPad camera to take an image of an object. Monty then moves a small patch over this image and tries to recognize the object.
 
 See the [monty_lab project folder](https://github.com/thousandbrainsproject/monty_lab/tree/main/monty_meets_world) for more details.
 
+
+
+# May, 2025 Robot Hackathon
```

Comment:
I was thinking of this project showcase page to eventually feature any project that uses Monty, also external ones so I would rather not structure it by how we organized this hackathon. Can you upload a separate video of the lego presentation and demo and link this here and then link the ultrasound video under the new ultrasound project headline?